assets/admin/editor-wizard/style.scss

```diff
@@ -0,0 +1,15 @@
+.sensei-editor-wizard {
+	& &__footer {
```

Comment:
Do we really need the extra `& ` here in these classes?

Reply:
Not really. I am just scared about "specificity" bugs in CSS hahaha. Changed here b32f0cefe